package.json

```diff
@@ -44,22 +42,22 @@
     "size:sb": "cd ./packages/smart-board && limit-size",
     "size:dw": "cd ./packages/data-wizard && limit-size",
     "size:auto-chart": "cd ./packages/auto-chart && limit-size",
-    "size:packages": "run-p size:*",
+    "size-packages": "run-p size:*",
     "build:ca": "cd ./packages/chart-advisor && npm run build",
     "build:ckb": "cd ./packages/ckb && npm run build",
     "build:li": "cd ./packages/lite-insight && npm run build",
     "build:sb": "cd ./packages/smart-board && npm run build",
     "build:dw": "cd ./packages/data-wizard && npm run build",
     "build:auto-chart": "cd ./packages/auto-chart && npm run build",
-    "build:packages": "lerna run build",
+    "build-packages": "lerna run build",
     "build": "npm run build:packages",
```

Comment:
`npm run build:packages` -> `npm run build-packages`

Reply:
any difference?

Reply:
> any difference?

You've replaced `build:packages` with `build-packages`. When run `build`, it will still execute `npm run build:packages`. But, there is no script `build:packages` now.

Reply:
oh...get